ant_plus_sensor/ant_plus_logger.js

```diff
@@ -7,7 +7,7 @@ const RollingAverage = require('./utils/average');
 const argumentParser = new ArgumentParser({
   version: '1.0.0',
   addHelp: true,
-  description: 'MHP DAS',
+  description: 'Ant-Plus MQTT Logger',
```

Comment:
Don't really care for this PR but so you know, Ant-Plus is usually stylised as Ant+